yacl/crypto/primitives/tpre/BUILD.bazel

```diff
@@ -0,0 +1,171 @@
+load("//bazel:yacl.bzl", "AES_COPT_FLAGS", "yacl_cc_binary", "yacl_cc_library", "yacl_cc_test")
+
+package(default_visibility = ["//visibility:public"])
+
+yacl_cc_library(
+    name = "kdf",
+    srcs = [
+        "kdf.cc",
+    ],
```

Comment:
Another style issue, sorry I forgot to comment earlier, we are using bazel beautifier to format all the `*.bazel` files, would you please check in the vscode market for [bazel extension](https://marketplace.visualstudio.com/items?itemName=BazelBuild.vscode-bazel)? You may also need to download the [bazel buildifier](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md)